src/Protocol/Smtp/NullConnection.php

```diff
@@ -0,0 +1,80 @@
+<?php
+declare(strict_types=1);
+
+namespace Genkgo\Mail\Protocol\Smtp;
+
+use Genkgo\Mail\Protocol\ConnectionInterface;
+
+/**
+ * Class NullConnection
+ * @package Genkgo\Mail\Protocol
+ */
+final class NullConnection implements ConnectionInterface
+{
+    private $receive;
+
+    public function addListener(string $name, \Closure $callback): void
+    {
+    }
+
+    public function connect(): void
+    {
+    }
+
+    public function disconnect(): void
+    {
+    }
+
+    public function send(string $request): int
+    {
+        $this->setReceive($request);
```

Comment:
My aversion to `set` is pretty big... :) I'd prefer something like `pushResponseToBuffer`.

Reply:
Done